apps/web-client/src/lib/db/cr-sqlite/customers.ts

```diff
@@ -291,3 +291,35 @@ export const markCustomerOrderAsReceived = async (db: DB, isbns: string[]): Prom
 		);
 	});
 };
+
+/**
+ * Marks customer order lines as collected when customer order lines are picked up.
+ * For each supplied ISBN, it updates the earliest unfulfilled customer order line
+ * (that has been received but not collected) with the current timestamp as collected date.
+ *
+ * @param {DB} db - The database connection instance
+ * @param {isbns[]} isbns - Array of supplier order line isbns that have been collected
+ * @returns {Promise<void>} A promise that resolves when all relevant customer orders are marked as collected
+ */
+
+export const markCustomerOrderAsCollected = async (db: DB, isbns: string[]): Promise<void> => {
```

Comment:
The `markCustomerOrderAsCollected` curently only takes ISBNs as parameter.

It has no way to know exactly what customer (and thus order line) it should mark as collected.

Either it takes customer id and isbn (but then if a single customer ordered the same book twice we might change "the other row" when users click on the button in the UI, since we have no way to differentiate the two rows) or a `rowid` for a specific `customer_order_line`.

Reply:
Customer order line actually has an `id` property so that seems the most straightforward way to handle this.